store/copr/mpp.go

```diff
@@ -231,6 +248,44 @@ func (m *mppIterator) handleDispatchReq(ctx context.Context, bo *tikv.Backoffer,
 	m.establishMPPConns(bo, req, taskMeta)
 }
 
+// NOTE: We do not retry here, because retry is helpless when errors result from TiFlash or Network. If errors occur, the execution on TiFlash will finally stop after some minutes.
+// This function is exclusively called, and only the first call succeeds sending tasks and setting all tasks as cancelled, while others will not work.
+func (m *mppIterator) cancelMppTasks() {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	killReq := &mpp.CancelTaskRequest{
+		Meta: &mpp.TaskMeta{
+			StartTs:     m.startTs,
+			TaskId:      0,
```

Comment:
The rest three lines can be deleted because we only care of the start ts

Reply:
ok